ONSdigital/eq-questionnaire-runner
# Make survey title generic [Bug]

### What is the context of this PR?
The survey title is currently set to ONS Business Survey even when no cookie exists. This PR changes it so the generic title, which is "ONS Surveys" which no cookie exist

### How to review 
- Locally run runner
- Start any survey, and remove the session using developer tools
- Refresh the page and it should return an error page with the title "ONS Surveys"
- Check if unit tests pass 

### Checklist
